spec/kitchen/driver/gce_spec.rb

```diff
@@ -117,7 +118,8 @@
         zone_name: 'europe-west1-a',
         google_key_location: '/path/to/foo.p12',
         google_json_key_location: '/path/to/bar.json',
-        preemptible: true
+        preemptible: true,
+        auto_restart: true
```

Comment:
I may be missing something, but per your [earlier comment](https://github.com/test-kitchen/kitchen-google/pull/22#issuecomment-167437410), it sounds like `preemptible` and `auto_restart` cannot both be true. Is that correct? If so, `auto_restart` must be `false` here. You can set `auto_restart` to `true` in the other case where `preemptible` is already `false` if you'd like.